### ISSUE TITLE
Auto-imports should go after the initial file comment

### ISSUE THREAD

**Issue by billti** [microsoft/qsharp#1770]
Auto-imports get added on line 1 currently. This is commonly a license header/copyright or file summary, and any added code should go after the initial comment block (if present). Example auto-import added on line 1 below.

<img width="763" alt="image" src="https://github.com/user-attachments/assets/f619f3f3-847b-485c-9ffb-8a10b689646f">

**Comment by minestarks** [comment]
Agreed that the experience isn't ideal here, but the "header" comment here is just a doc comment that applies to the first `import` item. Our samples seem to have missed this. We don't have the notion of a file-level "header" comment in the compiler - how would the auto-import feature be able to tell if this is a real doc comment or a "header" comment? 

```qsharp
/// I'm a doc comment
operation Foo() : Unit {
}
```

We wouldn't want to insert an `import` between `Foo` and its doc comment here.


We could probably fix our samples to use regular `//` and simply fix `get_first_non_whitespace_in_source` to skip all regular comments at the beginning of the file (just like we skip the `//qsharp` magic comment)
